packages/flutter/lib/fix_data.yaml

```diff
@@ -14,6 +14,453 @@
 
 version: 1
 transforms:
+  # Changes made in https://github.com/flutter/flutter/pull/94156
+  - title: "Migrate to 'ColorScheme.background'"
+    date: 2021-11-26
+    element:
+      uris: [ 'material.dart' ]
+      field: 'backgroundColor'
+      inClass: 'ThemeData'
+    changes:
+      - kind: 'rename'
+        newName: 'colorScheme.background'
+
+  # Changes made in https://github.com/flutter/flutter/pull/94156
+  - title: "Migrate to 'ColorScheme.background'"
+    date: 2021-11-26
+    element:
+      uris: [ 'material.dart' ]
+      method: 'copyWith'
+      inClass: 'ThemeData'
+    oneOf:
+    - if: "backgroundColor != '' && primarySwatch == '' && colorScheme == ''"
+      changes:
+      - kind: 'addParameter'
+        index: 56
+        name: 'colorScheme'
+        style: optional_named
+        argumentValue:
+          expression: 'ColorScheme.fromSwatch().copyWith(background: {% backgroundColor %})'
+          requiredIf: "backgroundColor != '' && primarySwatch == '' && colorScheme ==''"
+      - kind: 'removeParameter'
+        name: 'backgroundColor'
+    - if: "backgroundColor != '' && primarySwatch != '' && colorScheme == ''"
+      changes:
+      - kind: 'addParameter'
+        index: 56
+        name: 'colorScheme'
+        style: optional_named
+        argumentValue:
+          expression: 'ColorScheme.fromSwatch(primarySwatch: {% primarySwatch %}).copyWith(background: {% backgroundColor %})'
+          requiredIf: "backgroundColor != '' && primarySwatch != '' && colorScheme == ''"
+      - kind: 'removeParameter'
+        name: 'backgroundColor'
+      - kind: 'removeParameter'
+        name: 'primarySwatch'
+    - if: "backgroundColor != '' && primarySwatch == '' && colorScheme != ''"
+      changes:
+      - kind: 'removeParameter'
+        name: 'colorScheme' # Remove to add back with modification
+      - kind: 'addParameter'
+        index: 56
+        name: 'colorScheme'
+        style: optional_named
+        argumentValue:
+          expression: '{% colorScheme %}.copyWith(background: {% backgroundColor %})'
```

Comment:
I think this is why an additional copyWith is being added.

Reply:
I was able to simplify the output a little bit, but couldn't find a way to avoid the additional `copyWith` in the case where both colors are present. I think it's still a fine change.